mvc/src/main/java/webmvc/org/springframework/web/servlet/mvc/tobe/AnnotationHandlerMapping.java

```diff
@@ -19,10 +21,40 @@ public AnnotationHandlerMapping(final Object... basePackage) {
     }
 
     public void initialize() {
-        log.info("Initialized AnnotationHandlerMapping!");
+        final Reflections reflections = new Reflections(basePackage);
+        final Set<Class<?>> controllers = reflections.getTypesAnnotatedWith(Controller.class);
+        for (final Class<?> controller : controllers) {
+            SingletonRegistry.registerInstance(controller);
+            processController(controller);
+        }
```

Comment:
```suggestion
        final Set<Class<?>> controllers = reflections.getTypesAnnotatedWith(Controller.class);
        SingletonRegistry.registerInstance(controllers);
        controllers.forEach(this::processController);
```

제가 코드를 기존처럼 할지, 위처럼할지 고민인데, 지토는 뭐가 더 낫다고 생각하시나요?

Reply:
오 상세한 의견 감사해요 👍👍
저도 이게 forEach로 처리가 될 것 같아서 시도해본건데 억지로 들여쓰기를 없애버린 느낌이 들더라구요
그래서 제가 forEach를 거의 사용을 안해오니까 눈에 안익어서 그런건지, 원래 잘 안쓰이는건지 모르겠어서 지토는 어떤지 궁금했어요